{"name": "test_add_user_invalid_email", "status": "failed", "statusDetails": {"message": "AssertionError: assert 210 == 400", "trace": "self = <tests.test_api.TestApi object at 0x10a9244d0>\napi_client = <api.api_client.ApiClient object at 0x10a98e9d0>\n\n    def test_add_user_invalid_email(self, api_client):\n        \"\"\"\n        Тестируется негативный сценарий, когда производится попытка добавить пользователя сo значением \"email\",\n        не проходящим под стандартную маску почты\n        Ожидаемый статус код - 400 \"Плохой запрос\"\n        АPI работает некорректно: пользователь с таким email добавляется в базу и возвращается статус-код 210\n        \"\"\"\n        username = 'cvbinokil'\n        result = api_client.reg_user(username, 'yandex.ru', 'rwrwerwerwr')\n        self.builder.del_user(username)\n>       assert result == 400\nE       assert 210 == 400\n\ntests/test_api.py:65: AssertionError"}, "description": "\n        Тестируется негативный сценарий, когда производится попытка добавить пользователя сo значением \"email\",\n        не проходящим под стандартную маску почты\n        Ожидаемый статус код - 400 \"Плохой запрос\"\n        АPI работает некорректно: пользователь с таким email добавляется в базу и возвращается статус-код 210\n        ", "start": 1590610209214, "stop": 1590610209262, "uuid": "5532f80a-97e8-4908-aec3-cefbeec92dc5", "historyId": "122205f9d5ac4126e6241f036f8f6533", "testCaseId": "3ae16f72d704c9968e4abbf56f41c5fb", "fullName": "tests.test_api.TestApi#test_add_user_invalid_email", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "MacBook-Air-Roman-2.local"}, {"name": "thread", "value": "20823-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}